doc/manual.md

```diff
@@ -490,6 +494,11 @@ All these functions currently require float arguments. The compiler will reject
 At this time, math.log requires two parameters, and the compiler will reject one parameter calls.
 So as a workaround, use math.ln in place of the one parameter math.log.
 
+math.modf returns two values, an integer and a float. One minor difference from Lua is that Lua 
+may return a float for the first value if some reason it cannot return an integer, such as if
+the result is NaN (which only exists in floating point). In Pallene, the first value will always be
+converted to an integer.
```

Comment:
I believe that this difference also applies to other functions that use `pushnumint` internally, including `ceil` and `floor`.

Reply:
You are right. I missed that. I just pushed a change to make ceil and floor return integers. I just did a direct cast to LUA_INTEGER instead of making a set of new inline functions just to call lua_numbertointeger. I think that will be sufficient in these cases since we can't really fall back to float anyway if it doesn't fit.

Reply:
One more push to include ceil/float in the manual discussion about the integer conversion.